docs/contributor/container.md

```diff
@@ -16,22 +16,19 @@ Parity builds and publishes a container image that can be found as `docker.io/pa
 ## Parity CI image
 
 Parity maintains and uses internally a generic "CI" image that can be used as a base to build binaries: [Parity CI
-container image](https://github.com/paritytech/scripts/tree/master/dockerfiles/ci-linux):
+container image](https://github.com/paritytech/scripts/tree/master/dockerfiles/ci-unified):
 
 The command below allows building a Linux binary without having to even install Rust or any dependency locally:
 
 ```bash
 docker run --rm -it \
     -w /polkadot-sdk \
     -v $(pwd):/polkadot-sdk \
-    paritytech/ci-linux:production \
+    paritytech/ci-unified:bullseye-1.75.0-2024-01-22-v20240222 \
     cargo build --release --locked -p polkadot-parachain-bin --bin polkadot-parachain
 sudo chown -R $(id -u):$(id -g) target/
 ```
 
-If you want to reproduce other steps of CI process you can use the following
-[guide](https://github.com/paritytech/scripts#gitlab-ci-for-building-docker-images).
-
```

Comment:
would be great to have some means to reproduce CI steps locally. Often it is not so obvious...

Reply:
Okay, I'll make a note about that.